addon/components/lazy-render-wrapper.js

```diff
@@ -120,46 +120,72 @@ export default Ember.Component.extend({
 	_shouldShowOnRender: false,
 
 	event: 'hover', // hover, click, focus, none
-	entryInteractionEvents: computed('event', function() {
+	_lazyRenderEvents: computed('event', function() {
 		// the lazy-render wrapper will only render the tooltip when
-		// the $parent element is interacted with. This CP defines which
-		// events will trigger the rendering. We always include focusin
-		// to keep the component accessible.
-		let entryInteractionEvents = ['focusin'];
+		// the $target element is interacted with. This CP defines which
+		// events will trigger the rendering. Unless event="none" we always
+		// include focusin to keep the component accessible.
 		let event = this.get('event');
+
+		if (event === 'none') {
+			return [];
+		}
+
+		let _lazyRenderEvents = ['focusin'];
+
 		if (event === 'hover') {
-			entryInteractionEvents.push('mouseenter');
+			_lazyRenderEvents.push('mouseenter');
 		} else if (event === 'click') {
-			entryInteractionEvents.push('click');
+			_lazyRenderEvents.push('click');
+		}
+
+		return _lazyRenderEvents;
+	}),
+
+	$target: computed('target', 'tetherComponentName', function() {
+		const target = this.get('target'); // #some-id
+		let $target;
+
+		if (target) {
+			$target = $(target);
+		} else if (this.get('tetherComponentName').indexOf('-on-component') >= 0) {
+			// TODO(Andrew) refactor this once we've gotten rid of the -on-component approach
+			// share the functionality with `onComponentTarget`
+			const targetViewElementId = this.get('parentView.elementId');
+			$target = $(`#${targetViewElementId}`);
+		} else {
+			$target = getParent(this);
 		}
 
-		return entryInteractionEvents;
+		return $target;
 	}),
 
+	_targetEventNameSpace: 'target-lazy-render-wrapper',
```

Comment:
Should this be a `const` in the file instead of a property on the component? That would make it truly private and would still avoid repetitions of the classname in the code.